README.md

```diff
@@ -1,111 +1,116 @@
-
-### Basic Schema
-
-The project provides an implementation of the `skeletor schema` command that is mostly functional. To fully implement this for your mixin, you simply need to provide a valid JSON schema. For reference, consult `pkg/skeletor/schema/schema.json`.
-
-### Basic Tests
-
-The project provides some very basic test skeletons that you can use as a starting point for building tests for your mixin.
-
-### Magefile
-
-The project also includes a [Magefile] that is used to build, test, and publish the mixin.
-
-### Publish
-
-You must set the `GITHUB_TOKEN` environment variable with your personal access token in order to use the default publish target.
-
-Publish uploads cross-compiled binaries of your mixin to a GitHub release.
-You must set the `PORTER_RELEASE_REPOSITORY` environment variable to your GitHub repository name, e.g. github.com/YOURNAME/YOURREPO.
-There is a placeholder in the Publish magefile target where you can set that value.
-
-Create a tag, for example `git tag v0.1.0`, and push it to your repository.
-Run `mage XBuildAll Publish` to build your mixin and upload the binaries to the github release for that tag.
-If the commit is not tagged, the release is named "canary".
-
-If you want to generate a mixin feed file (atom.xml), edit the Publish magefile target, uncomment out the rest of the function, and set the `PORTER_PACKAGES_REMOTE` environment variable to a repository where the atom.xml file should be committed.
-For example, Porter uses github.com/getporter/packages for publishing our mixin feed.
-
-[Magefile]: https://magefile.org
+# Nomad Mixin for Porter
+
+This is a [Porter](https://porter.sh) mixin for interacting with [Nomad](https://www.nomadproject.io/).
+
+## Installation
+
+todo
+
+## Mixin Syntax
+
+In your porter.yaml file, add the nomad as a mixin:
+
+```yaml
+mixins:
+  - nomad:
+      ...
+```
+
+The Mixin supports the following global settings, which will apply to all your nomad actions, unless overridden in the action
+itself:
+
+```yaml
+mixins:
+  - nomad:
+      address: ""
+      region: ""
+      namespace: ""
+      httpAuth: ""
+
+      # tls
+      caCert: ""
+      caPath: ""
+      clientCert: ""
+      clientKey: ""
+      tlsServerName: ""
+      tlsSkipVerify:
+
+      # acl
+      token: ""
+```
+
+To use the mixin in an Install/Upgrade/Uninstall step, add the "jobs" block to your porter.yaml file (using install here
+an example):
+
+```yaml
+install:
+  - nomad:
+      jobs:
+        - path: nomad/pytechco-redis.nomad.hcl
+        - path: nomad/pytechco-web.nomad.hcl
+        - path: nomad/pytechco-setup.nomad.hcl
+        - dispatch: pytechco-setup
+          meta:
+            budget: "200"
+        - path: nomad/pytechco-employee.nomad.hcl
+```
+
+The job step supports the following 3 actions which can be specified as a list item in the `jobs:` block:
```

Comment:
It looks like some mixins support the equivalent of Nomad's [`alloc exec`](https://developer.hashicorp.com/nomad/docs/commands/alloc/exec) (or other allocation commands like reading logs or files). That might be something worth doing for future iterations.

Reply:
Great point, thanks!